src/inference/src/ie_core.cpp

```diff
@@ -732,9 +736,15 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
 
         std::lock_guard<std::mutex> lock(pluginsMutex);
         auto deviceName = pluginName;
+        if (deviceName == ov::DEFAULT_DEVICE_NAME)
+            deviceName = "AUTO";
         auto it = pluginRegistry.find(deviceName);
         if (it == pluginRegistry.end()) {
-            IE_THROW() << "Device with \"" << deviceName << "\" name is not registered in the InferenceEngine";
+            if (pluginName == ov::DEFAULT_DEVICE_NAME)
+                IE_THROW() << "No device is provided, so AUTO device is used by default, which failed loading. Should "
```

Comment:
Please remove the second sentence

Reply:
As myshevts mentioned in previous PR https://github.com/openvinotoolkit/openvino/pull/8494#discussion_r757304345, maybe this exception message is needed for the users who didn't specify  the device name.